src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -100,15 +111,13 @@ def _get_all_property_names_and_resource_ids_one_resource(
     resource: etree._Element, prop_dict: dict[str, list[str]]
 ) -> dict[str, list[str]]:
     for prop in resource.iterchildren():
-        match prop.tag:
-            case "bitstream" | "iiif-uri":
-                pass
-            case _:
-                prop_name = prop.attrib["name"]
-                if prop_name in prop_dict:
-                    prop_dict[prop_name].append(resource.attrib["id"])
-                else:
-                    prop_dict[prop_name] = [resource.attrib["id"]]
+        if prop.tag in KNORA_BASE_PROPERTIES:
```

Comment:
Could replace this with a default dict? I think it would make things clearer.

Reply:
Yes, was possible without adding too much diff: https://github.com/dasch-swiss/dsp-tools/commit/6104f847a6f25c703e567449bd13c2d466696c4a